### DISCUSSION TITLE
Looking for a clarification for "not splitting on -r"

### DISCUSSION THREAD

**Discussion by evgenyfadeev** [aerickt/lapwing-for-beginners#31] [Q&A]
In the chapter 15 test there is:

redirect - RAOE/TKAOEU/REBGT

the final syllable starts with "R". The IPA transcription is - /ˌɹidɝˈɛkt/ - is R altered enough here by the preceding vowel to count as a vowel according to the rule in the chapter 15?

searching in the steno dictionary file gives more examples
"HA/RAFPLT": "harassment",  "/hɝˈæsmənt/",
"STAOEU/RAOEPB": "styrene",     "/ˈstaɪˌɹin/"
SKWRAOEU/RAEUT": "gyrate",   "/ˈdʒaɪˌɹeɪt/"

these I found in 2 minutes of inspection within ~4000 matches - so there must be more.

The example in CH 15 is "car" - on this kind of sound supposedly we should not split before R,, the IPA is - "/ˈkɑɹ/"

R is using the same IPA symbol as in styrene - so I guess there should not be split on R and yet it is in the lapwing-base.json ?
Is it possible to describe in more detail, where break on R is allowed and where it isn't?

**Comment by aerickt** [comment]
If the R is pronounced more like a consonant, then it should generally be treated like a consonant and can start a stroke. If the R alters the vowel sound, then it should be kept with the previous vowel.
Maybe these examples can help (try listening to the General North American pronunciations):
* percent `PER/SEPBT` (vowel)
  * https://en.wiktionary.org/wiki/percent
* parameter `PA/RA/PHE/TER` (consonant)
  * https://en.wiktionary.org/wiki/parameter
* car `KAR` (vowel)
  * https://en.wiktionary.org/wiki/car
* careen `KA/RAOEPB` (consonant)
  * https://en.wiktionary.org/wiki/careen

I'm not sure about this explanation, but I believe it's to do with stress. A word can be split at the primary or secondary stress right before the R consonant.
* /hə**ˈ**ɹæs.mənt/
* /ˈstaɪ**ˌ**ɹin/
* /ˈdʒaɪ**ˌ**ɹeɪt/
* /pə**ˈ**ɹæm.ɪ.tə/
* /kə**ˈ**ɹiːn/

**Reply by evgenyfadeev** [reply]
There seem to be some correlation with the stress. What also helped me is to think how the speakers of British English pronounce "r" - where they don't pronounce the r (when the word is isolated - the linked document below mentions that R is pronounced in phrase "car engine" but not in word "car" alone) - it is the "r vowel" (this concept I think might be a bit tricky for the North American English speakers as they almost always pronounce the "r" - there is a term for that - American English pronunciation is classified as "rhotic")

I've looked up the rules on silent r in English and found this:

https://pronunciationstudio.com/silent-r-british-pronunciation/

Speaking of the word "forensic" again - here the pronunciation of R is not different from that in "harassment" and according the "british r" rule - R is pronounced (as a consonant) if it comes before a vowel, so following this logic - splitting fo/ren/sic or even fo/rens/ic might be valid syllabic splittings (-ic would be a syllable, coincidental with -ic suffix).

Perhaps it might be useful to refer the learners to the "british R" pronunciation rule to decide whether to split or not before the R?

Alternatively - if there are very few examples where splitting on R is the only option - the syllabic split on R might be just forbidden and the exceptions listed explicitly.

Or indeed, as you suggest if it were all about the stress - then it would help explain the rule without diving into the British pronunciation rules.

Leaving a link here, there is a concept of "r-colored vowel" https://en.wikipedia.org/wiki/R-colored_vowel

**Reply by evgenyfadeev** [reply]
It might be that "consonant r" is there only if the "r" is followed by a vowel - and it is in line with the British pronunciation of R.

Conversely, if r is followed by a consonant or is the final letter of the word - this would be a "vowel r".

If this were true, it would be quite simple to formalize the "split on R" rule, because the rule would be based on orthography.

**Reply by evgenyfadeev** [reply]
@aerickt I've been investigating this and so far it seems that there is a slight correlation on when there is a pre-r split - if the stress falls on a vowel right after the "R".

So for forensic, I'd split fo/ren/sic at least as an additional option.

If stress falls immediately before the R, there is almost never a split pre-r (the only exceptions so far are virulelent SRAOEU/RAOU/LENT), environment (maybe it has to do with the ai sound), however word naira is not split before R, although TPHAOEU/RA seems like a valid outline.

Perhaps sometimes the pre-r split could be optionally used if neigher pre-r nor post-r vowel is stressed. That is both with and without pre-r split oulines could perhaps be used.

This only applies to the situation when R has a vowel on both sides, if there is a consonant on the left or right, there is no ambiguity for the split - consonant before R - no split, after R - split.

The concept of "vowel R" I find not easy to understand. In the American English "r" is always pronounced, unlike in British, besides there is no such official concept as "vowel R", neither there is an IPA symbol for it. Even if one argues that there are "R-flavored vowels", it might be hard to hear the difference between that and the version written with vowel and the upside-down r ipa symbols separately - I personally fail to hear the difference.

The second aspect - the vowel modification before R is clear partially to me, I definitely hear the difference between A in "cat" and "car", I in "fin" and "fir", or U in "fun" and "furnace", but not AI in "pay" and "pair", or E "benny" and "berry". That is in the latter two cases I find the vowels sound the same with and without the "r".

That is - if there were a way to explain the "split on r" based on simpler observables - such as stress and presence/absence of vowels around the r, perhaps it might be much easier to grok the rule.

**Reply by aerickt** [reply]
I'm sorry, I don't think there is anything else I can say to help you here. It seems our understanding of the English language and possibly our accents differ too much. I would suggest just adding your own `[AOEU]/R` style splits whenever you come across issues.

**Reply by evgenyfadeev** [reply]
No problems, I'm not contradicting your theory that much btw, the stress-based method mostly does not deviate from the "vowel/consonant R". It's just something I can explain to myself as all R-s sound the same to me in the American English.

The hardest part is when there is a vowel on both sides of the R and the stress is not before the R - that's where most of the deviations between the two methods are.

**Comment by aerickt** [comment]
Not sure if you've already tackled this, but I autogenerated a dictionary with a crude find and replace script. Might be helpful; let me know if you end up trying it out.
[lapwing-alternative-r.json](https://github.com/user-attachments/files/17690260/lapwing-alternative-r.json)

I may create another discussion post sharing the script and my other thoughts on the subject. I think this could definitely be a viable add on dictionary for Lapwing. Unfortunately, to merge this with the main dictionary and include it in the main docs would require a lot of changes. If this dictionary is helpful, my current plan is to just add an appendix and a note referencing it in chapter 15.

**Reply by evgenyfadeev** [reply]
I'll definitely try it, thanks! Currently I'm on the 200 most frequent words and it will take some time. Also started a bit in the typey-type project.
